ui/src/app/shared/workflow/node/hook/form/hook.form.html

```diff
@@ -64,6 +76,43 @@ <h3>{{ 'workflow_node_hook_form_config' | translate }}</h3>
                         </div>
                     </div>
                 </ng-container>
+                <ng-container *ngIf="isOutgoing && _hook.config && _hook.model.name == 'Workflow'">
```

Comment:
strict equals ===

Reply:
and check _hook.model ? if it is relevant

Reply:
fixed